pandas/_libs/tslibs/offsets.pyx

```diff
@@ -1903,30 +1913,44 @@ cdef class YearOffset(SingleConstructorOffset):
         return shifted
 
 
-cdef class BYearEnd(YearOffset):
+cdef class YearBegin(YearOffset):
```

Comment:
are you re-ordering the classes in this file?  or is git just mangling the diff?